app/views/volunteers/_manage_cases.erb

```diff
@@ -22,9 +22,9 @@
                   <td><%= assignment.casa_case.decorate.transition_aged_youth %></td>
                   <td>
                     <% if @volunteer.active? && assignment.casa_case.active? && assignment.active? %>
-                      Volunteer is <span class="badge badge-success text-uppercase display-1">Active</span>
+                      Volunteer is <span class="badge badge-success text-uppercase display-1 text-dark">Active</span>
```

Comment:
@Daniel-Penaloza We also want to utilize [Bootstrap's badging](https://getbootstrap.com/docs/5.0/components/badge/#background-colors) for background colors. Can you update the code to use that too?

Reply:
Sure @schoork, it's done ;)